backend-docker/pier/db/db.go

```diff
@@ -0,0 +1,551 @@
+package db
+
+import (
+	"bytes"
+	"database/sql"
+	"fmt"
+	"github.com/EUDAT-GEF/GEF/backend-docker/pier/internal/dckr"
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/pborman/uuid"
+	"gopkg.in/gorp.v1"
+	"log"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const GefSrvLabelPrefix = "eudat.gef.service." // GefSrvLabelPrefix is the prefix identifying GEF related labels
+const GorpVersionColumn = "Revision"           // Column in a table used to keep an internal version number of GORP
+const SQLiteDataBasePath = "gef_db.bin"
+
+// VolumeID contains a docker volume ID
+type VolumeID dckr.VolumeID
+
+// Db is used to keep DbMap
+type Db struct{ gorp.DbMap }
+
+// Job stores the information about a service execution (used to serialize JSON)
```

Comment:
I would say that Job, Service and the main datatypes should be in `pier`, not embedded here. In this way they are part of the official API.